### ISSUE TITLE
Change default port

### ISSUE THREAD

**Issue by profy12** [O-clock-Dev/teleporter-compose#72]
Default port 80 is in conflict with an eventual already installed http aplication. So Application will not start and cause a bad experience to users.

We should bind by default another port like 1337, and redirect eventually port 80 to 1337 if port 80 is free in a second time.

**Comment by clementgoclock** [comment]
Still a thing ?

**Comment by profy12** [comment]
Yes we should open on another port than 80, maybe 4242, and make a no citical redirect from port 80 to this port.
